pkg/ingester/errors_test.go

```diff
@@ -248,8 +248,8 @@ func TestTooBusyError(t *testing.T) {
 
 func TestNewCircuitBreakerOpenError(t *testing.T) {
 	remainingDelay := 1 * time.Second
-	expectedMsg := fmt.Sprintf("circuit breaker open with remaining delay %s", remainingDelay.String())
-	err := newCircuitBreakerOpenError(remainingDelay)
+	expectedMsg := fmt.Sprintf("circuit breaker open on aaa request type with remaining delay %s", remainingDelay.String())
+	err := newCircuitBreakerOpenError("aaa", remainingDelay)
```

Comment:
I thought `aaa` was a typo first.

I think talking about request type in test name is too verbose, especially since they all have a request type now.
```suggestion
	expectedMsg := fmt.Sprintf("circuit breaker open with remaining delay %s", remainingDelay.String())
	err := newCircuitBreakerOpenError("foo", remainingDelay)
```

Reply:
Oh, I misread this, sorry.